cmd/generate/config/rules/curl.go

```diff
@@ -75,3 +76,98 @@ nc -u -l 41234`,
 	}
 	return utils.Validate(r, tps, fps)
 }
+
+// https://curl.se/docs/manpage.html#-H
+func CurlHeaderAuth() *config.Rule {
+	// language=regexp
+	authPat := `(?i)(?:Authorization:[ \t]?(?:Basic[ \t]([a-z0-9+/]{8,}={0,3})|(?:Bearer|Token)[ \t]([\w=@.+/-]{8,})|([\w=.+/-]{8,}))|(?:ApiKey|Token|X-API-KEY):[ \t]?([\w=@.+/-]{8,}))`
+	r := config.Rule{
+		RuleID: "curl-auth-header",
+		// TODO: Description: "",
+		Regex: regexp.MustCompile(
+			fmt.Sprintf(`\bcurl\b(?:.*?|.*?(?:[\r\n]{1,2}.*?){1,5})[ \t\n\r](?:-H|--header)[ =](?:"%s"|'%s')(?:\B|\s|\z)`, authPat, authPat)),
+		Entropy:  2,
```

Comment:
How did you arrive at this number? Could we kick it up to 3 or 3.5 to further filter out FPs?

Reply:
I opted for a lower number because this case doesn't get detected (it's 2.91):
```
`curl -XPOST http://localhost:8080/api/tasks -H "Content-Type: application/x-www-form-urlencoded; charset=UTF-8" -H "Token: 3fea6af1349166ea" -d "content=hello-curl"`, // token
```

I've bumped it up to 2.75.